lib/accumulate_distribute/index.js

```diff
@@ -19,6 +19,7 @@ const genPreview = require('./meta/gen_preview')
 const declareEvents = require('./meta/declare_events')
 const declareChannels = require('./meta/declare_channels')
 const getUIDef = require('./meta/get_ui_def')
+const confirmOrderSubmit = require('./meta/confirm_order_submit')
```

Comment:
looks like we are editing acc/dis here, shouldn't it be twap?

Reply:
PR separated for acc/dist and twap.